### DISCUSSION TITLE
Can't get koreader work properly. Stucked at usbms session screen.

### DISCUSSION THREAD

**Discussion by pyromaker** [koreader/koreader#9677] [Q&A]
Hi, I got a kobo Libra 2 device yesterday and of course the first thing I did was to install koreader.
Before that I had a kobo glo hd that broke and the koreader there was working like a charm.

It's been years when I installed koreader in my glo hd so I couldn't remember which guide I followed to install it back then.

After some reading I tried to install the latest version of koreader with this method
Alternative Manual Installation Method based on NickelMenu:
https://github.com/koreader/koreader/wiki/Installation-on-Kobo-devices

I somehow managed to do it but I have the feeling that it doesn't work properly. Some kind of errors popped up when I run koreader  for the first time.

Mostly of 99% of the time I used the Newsdownloader and in my glo hd it was working very good.

In my new device, when I press Menu-News downloader (RSS/ATOM) nothing happens (I really liked that setting to avoid plugging all the time the ereader with my pc to play with the news settings file). 

So then I tried to connect the device with my pc to "play" with the news settings file and now it's stucked at the "Starting USBMS session" with the brightness on a and I can't do nothing.
When I tried to connect my device with the PC, this starting usbms session screen popped up as normally but my pc didn't identify it, so i unplugged the usb cable and now it's stucked there with the brightness on... My question is, is there something I could try to fix it? 

I am very sad at this moment :(

EDIT: Ok so my device is not stuck anymore. Tried for the 10th time to power off the device and it worked.
About the newsdownloader, Am I doing something wrong? Try to reinstall it the koreader?
Thanks.

**Comment by NiLuJe** [comment]
Try the install script instead?

Keep in mind the Libra 2 is a crashy mess, so any and all weirdness *may* be laid at its feet.

**Reply by pyromaker** [reply]
Thanks for the reply.

Well I tried to install with the script method and I got a message "file path not found" so I tried the other way

Maybe I'll try again.
Is it safe to Uninstall the koreader by deleting the folder koreader?
